doc/man3/SSL_CTX_set_options.pod

```diff
@@ -200,6 +200,17 @@ this option is set or not CCS messages received from the peer will always be
 ignored in TLSv1.3. This option is set by default. To switch it off use
 SSL_clear_options(). A future version of OpenSSL may not set this by default.
 
+=item SSL_OP_NO_ANTI_REPLAY
+
+By default, when a server is configured for early data (i.e. max_early_data > 0),
```

Comment:
Since this is a nit-fest already, comma after "i.e.".

Reply:
Done